### ISSUE TITLE
Displaying Buses

### ISSUE THREAD

**Issue by Patschke** [Klizzy/MMM-Vrr#5]
Am I missing something, or is there no option to display buses? The Readme says:

> Displays the next departure times of Trains, subway and Buses from any city and station in the german federal state North Rhine-Westphalia (VRR).

But I can't see any Buses beeing shown. Is there a config option I missed?

**Comment by Klizzy** [comment]
Normally it displays busses, if they are also seen at vrr.de
This modules uses the same api, so if they aren't seen there they will not be displayed on the MagicMirror. 
If you provide me a City + Station i could check if its a bug

**Comment by Patschke** [comment]
If I recall correctly it was a bug created by different handling of City + Station name combinations for some of our bus stops (e.g. separated by / or ,). The issues were probably on either Kaiserslautern Uni Ost or Uni West (or maybe Kaiserslautern Uni Süd, but I think we never cared much about that bus stop). 

We've fixed the issues in our local fork, at https://gitlab.rlp.net/fsr_physik_tu_kl/getraenke/mmm-vrr/. If those changes do not break anything else (I have to admit that I did no in depth testing, as I did not care much about any other stations), you may want to merge them.

**Comment by Klizzy** [comment]
Sorry for the late reply. Could you create an pull request into this repo? After that i could test and merg it

**Comment by Patschke** [comment]
As we have our repo not on github, but on a gitlab instance hosted elsewhere, it is not possible to do a direct PR (I'd have to create a clone of the repo first). However, you should be able to check out and merge our repo just fine (`git remote add fsphi https://gitlab.rlp.net/fsr_physik_tu_kl/getraenke/mmm-vrr.git` and `git fetch fsphi` and `git merge fsphi/master`). 

If that doesn't work for you for any reason, I could also create a clone here on github - it's just more work than directly merging our repo.

**Comment by Klizzy** [comment]
thanks for the fast reply! 

So just for clarity:
The main reason behind this change was, to also support the `VRN` area? A quick look into the changes suggests, that some API and response parameter have mostly changed.

If that's the case i can test and extend your changes, so that `VRN` can be added to the official supported areas

**Comment by Patschke** [comment]
As far as I remember that's correct. We were interested in unsing this for the bus stops at our university, and some of the responses from the backend did not work/some patterns did not match. However, I can't tell if those patterns are just different from location to location and a more general handling is required. 

By the way: Thanks for the plugin, we have it in use for some years now without any trouble.

**Comment by Klizzy** [comment]
Yeah thanks for using my plugin! Its always nice to hear if it's useful for someone 😄 

One last thing:
Could you provide me your Request URL or one example Station and City combination? I only get mixed results, with api calls like that `https://vrrf.finalrewind.org/Kaiserslautern/Uni-Ost.json?frontend=json&backend=efa.VRN`, `uni-ost,kaiserslautern.json` or other combinations.

If i have your exact call i can verify that my implementation is still working with your needed route

**Comment by Patschke** [comment]
I don't really care what the exact api call is that the module executes, as long as it works - so I think it is somewhat more reasonable to give you the MMM-Vrr part of our config? As long as that works reasonable well I'm happy. 

```
		{
			module: "MMM-Vrr",
			header: "Fahrplan Uni Ost",
			position: "right",
			config: {
				city: "Kaiserslautern",
				station: "Uni Ost",
				numberOfResults: 5,
				displayTimeOption: "time",
				displayType: "detail",
                                showTrack: false,
				showDelay: false,	
			},
		},
		{
			module: "MMM-Vrr",
			header: "Fahrplan Uni West",
			position: "right",
			config: {
				city: "Kaiserslautern",
				station: "Universität West",
				numberOfResults: 5,
				displayTimeOption: "time",
				displayType: "detail",
                                showTrack: false,
				showDelay: false,
			},
		},
```

**Comment by Klizzy** [comment]
it comes down to the same thing. Now i can verify the functionality, thanks!
